packages/date/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Bundle type definitions.
```

Comment:
It could have a section like:

```md
### New Features
```

The same applies to the `api-fetch` package.